core/tauri/src/webview/mod.rs

```diff
@@ -1044,17 +1044,47 @@ fn main() {
   }
 
   fn is_local_url(&self, current_url: &Url) -> bool {
-    self
-      .manager()
-      .get_url()
-      .make_relative(current_url)
-      .is_some()
-      || {
-        let protocol_url = self.manager().protocol_url();
-        current_url.scheme() == protocol_url.scheme()
-          && current_url.domain() == protocol_url.domain()
-      }
+    // if from `tauri://` custom protocol
+    ({
+      let protocol_url = self.manager().protocol_url();
+      current_url.scheme() == protocol_url.scheme()
+      && current_url.domain() == protocol_url.domain()
+    }) ||
+
+    // or if relative to `distDir` or `devPath`
+      self
+          .manager()
+          .get_url()
+          .make_relative(current_url)
+          .is_some()
+
+      // TODO: what use-case is covered by this?
       || (cfg!(dev) && current_url.domain() == Some("tauri.localhost"))
```

Comment:
@lucasfernog what use-case is being covered by this check?

Reply:
IIRC it's for mobile.. should've documented it my bad

Reply:
won't this be covered by the `tauri:://` case as well?